src/app/intro/intro.component.tns.html

```diff
@@ -1,15 +1,15 @@
-<ScrollView>
-    <Stacklayout>
-        <Label text="Examples" style="font-size: 32; margin: 10"></Label>
-        <Button kirby-button text="Button" [nsRouterLink]="['/examples/button']" class="btn btn-primary"></Button>
-        <Button kirby-button text="Card" [nsRouterLink]="['/examples/card']" class="btn btn-primary"></Button>
-        <Button kirby-button text="List" [nsRouterLink]="['/examples/list']" class="btn btn-primary"></Button>
-        <Button kirby-button text="Grid" [nsRouterLink]="['/examples/grid']" class="btn btn-primary"></Button>
-        <Button kirby-button text="Fonts" [nsRouterLink]="['/examples/fonts']" ></Button>
-        <Button kirby-button text="Charts" [nsRouterLink]="['/examples/chart']" ></Button>
-        <Button kirby-button text="Spinner" [nsRouterLink]="['/examples/spinner']" ></Button>
-        <Label text="{N} Only Components:" style="font-size: 18; margin: 10"></Label>
-        <Button kirby-button text="Doughnut Chart" [nsRouterLink]="['/examples/nativescript-only/doughnut-chart']" class="btn btn-primary"></Button>
-        <Button kirby-button text="Line Chart" [nsRouterLink]="['/examples/nativescript-only/line-chart']" class="btn btn-primary"></Button>
-    </Stacklayout>
-</ScrollView>
+<Stacklayout>
```

Comment:
Yaaaa.. lets not remove the scroll view here. On a iPhone 6 i can no longer scroll in the demo app and get to the bottom menu item.

![image](https://user-images.githubusercontent.com/46445487/53735158-5ee33880-3e86-11e9-9731-c68c262b3eb5.png)